internal/config/audit.go

```diff
@@ -49,8 +49,14 @@ func (c *AuditConfig) validate() error {
 		return errors.New("file not specified")
 	}
 
-	if c.Sinks.Webhook.Enabled && c.Sinks.Webhook.URL == "" {
-		return errors.New("url not provided")
+	if c.Sinks.Webhook.Enabled {
+		if c.Sinks.Webhook.URL == "" && len(c.Sinks.Webhook.Templates) == 0 {
+			return errors.New("url or template(s) not provided")
+		}
+
+		if c.Sinks.Webhook.URL != "" && len(c.Sinks.Webhook.Templates) > 0 {
+			return errors.New("only one of url or template(s) allowed")
+		}
```

Comment:
still seems to be missing a test

Reply:
@markphelps This one was tough. I'll try to see how to craft a test for it. Essentially, it is mostly due to the weirdness of representing an array of objects as an environment variable.